Dockerfile

```diff
@@ -76,7 +76,7 @@ RUN python -m venv /venv
 ENV PATH=/venv/bin:$PATH
 # Install up-to-date versions of installation tools, for the benefits of
 # packages not using PEP 517/518.
```

Comment:
This comment needs to be updated to explain why jinja etc are installed: it's for compiling the C++ bindings of spead2.